quaterion/loss/multi_objective_loss.py

```diff
@@ -0,0 +1,22 @@
+from typing import Optional
+
+import torch
+
+
+class MultiObjectiveLoss(torch.nn.Module):
+    def __init__(self, losses: list, weights: Optional[list] = None):
```

Comment:
I think, in order to improve expressivity, that would be better to include data type stored in the list such as `weights: Optional[List[Tensor]]`.